WalletLibrary/WalletLibrary/LibraryConfiguration.swift

```diff
@@ -17,22 +17,25 @@ class LibraryConfiguration {
     
     let verifiedIdEncoder: VerifiedIdEncoding
     
-    let correlationHeader: VerifiedIdCorrelationHeader?
+    let networking: LibraryNetworking
     
     let identifierManager: IdentifierManager
     
     let previewFeatureFlags: PreviewFeatureFlags
 
     init(logger: WalletLibraryLogger = WalletLibraryLogger(),
          mapper: Mapping = Mapper(),
-         correlationHeader: VerifiedIdCorrelationHeader? = nil,
+         networking: LibraryNetworking? = nil,
          verifiedIdDecoder: VerifiedIdDecoding = VerifiedIdDecoder(),
          verifiedIdEncoder: VerifiedIdEncoding = VerifiedIdEncoder(),
          identifierManager: IdentifierManager? = nil,
-         previewFeatureFlags: PreviewFeatureFlags = PreviewFeatureFlags()) {
+         previewFeatureFlags: PreviewFeatureFlags = PreviewFeatureFlags()) 
+    {
         self.logger = logger
         self.mapper = mapper
-        self.correlationHeader = correlationHeader
+        self.networking = networking ?? WalletLibraryNetworking(urlSession: URLSession.shared,
+                                                                logger: logger,
+                                                                correlationHeader: nil)
```

Comment:
doesn't correlationHeader need to be injected here?

Reply:
The correlation header is injected int the WalletLibraryNetworking in the layer above so in the VerifiedIdClientBuilder, so there would not be a case where it would be injected here if WalletLibraryNetworking is nil.